scss/_utilities/_mediaqueries.scss

```diff
@@ -0,0 +1,43 @@
+/* 
+* Media Query Mixin  
+* Containes logic to easily remove media queries for
+* desktop only or mobile only projects
+* Examples:
+* 1. @include mq( 300px ) { } (min-width)
+* 2. @include mq( $max: 300px ) { } (max-width)
+* 3. @include mq( 400px, 800px ) { } (both min + max-width)
+*/
+$media-queries: true;
+$desktop-styles: true;
+
+@mixin mq($min: null, $max: null) {
+	@if $min == null and $max == null {
+		@error 'Please provide a value for $min and/or $max';
```

Comment:
According to the style guide, use double quites instead of single quotes: https://contribute.jquery.org/style-guide/css/#formatting